diff --git a/index.d.ts b/index.d.ts
index 847c7fd73e085e1e1eef4c7fe149c941859c88e2..9a4289c0cae25dacfb1ad101f743ccc2697a2bc9 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -1,4 +1,3 @@
 export * from "./src/types.js";
-export * from "./src/exos/cosmos-interchain-service.js";
 export * from "./src/typeGuards.js";
 //# sourceMappingURL=index.d.ts.map
diff --git a/index.js b/index.js
index 09afecb813dd5350f93cb1d482575efa84af85b7..ad5704a3169c12170980cac9865a8f325dea66cd 100644
--- a/index.js
+++ b/index.js
@@ -3,5 +3,4 @@
 /// <reference types="@agoric/zoe/exported" />
 
 export * from './src/types.js';
-export * from './src/exos/cosmos-interchain-service.js';
 export * from './src/typeGuards.js';
