diff --git a/lib/bn.js b/lib/bn.js
index 3a4371ea29688d216480d68af653c52b8b7beb10..5da31f6f1821c784c5b4aee288997cedc6a25ec3 100644
--- a/lib/bn.js
+++ b/lib/bn.js
@@ -2533,7 +2533,7 @@
       for (var i = 0, im = 1; (x.words[0] & im) === 0 && i < 26; ++i, im <<= 1);
       if (i > 0) {
         x.iushrn(i);
-        while (i-- > 0) {
+        while ([i--][0] > 0) {
           if (A.isOdd() || B.isOdd()) {
             A.iadd(yp);
             B.isub(xp);
@@ -2547,7 +2547,7 @@
       for (var j = 0, jm = 1; (y.words[0] & jm) === 0 && j < 26; ++j, jm <<= 1);
       if (j > 0) {
         y.iushrn(j);
-        while (j-- > 0) {
+        while ([j--][0] > 0) {
           if (C.isOdd() || D.isOdd()) {
             C.iadd(yp);
             D.isub(xp);
@@ -2601,7 +2601,7 @@
       for (var i = 0, im = 1; (a.words[0] & im) === 0 && i < 26; ++i, im <<= 1);
       if (i > 0) {
         a.iushrn(i);
-        while (i-- > 0) {
+        while ([i--][0] > 0) {
           if (x1.isOdd()) {
             x1.iadd(delta);
           }
@@ -2613,7 +2613,7 @@
       for (var j = 0, jm = 1; (b.words[0] & jm) === 0 && j < 26; ++j, jm <<= 1);
       if (j > 0) {
         b.iushrn(j);
-        while (j-- > 0) {
+        while ([j--][0] > 0) {
           if (x2.isOdd()) {
             x2.iadd(delta);
           }
